APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -436,6 +440,24 @@ - (UITableViewCell *) tableView:(UITableView *)tableView cellForRowAtIndexPath:(
     return cell;
 }
 
+- (void) setEditing:(BOOL) isEditing
+{
+    [super setEditing:isEditing];
+    
+    NSArray* visibleCells = [self.tableView visibleCells];
+    for (UITableViewCell* cell in visibleCells) {
```

Comment:
This feels hacky and hard to maintain (and we still will need to maintain AppCore at least for a little while, despite our best efforts). Does a simple call to [self.tableView reloadData] not work here? If not, can you do it by overriding willTransitionToState: or didTransitionToState: on the APCSegmentedTableViewCell?